README.md

```diff
@@ -76,6 +76,9 @@ See [here](#continuous-integration-ci) for details.
 - Simple
   - Specify only an image name
   - See [Quick Start](#quick-start) and [Examples](#examples)
+- Fast
+  - The first scan will finish within 10 seconds (depending on your network)
+  - Most scanners take a while (~10 minutes) to fetch vulnerability information on the first run
```

Comment:
comparing to clair where the server fetches the info it's true but with clair people will probably keep the server around to mitigate this first run cost. this is another benefit of trivy that it's stateless and you can just run it from scratch every time and throw it away after